src/splitInteger.test.js

```diff
@@ -4,18 +4,18 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(8, 4)).toEqual([2, 2, 2, 2]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
+  expect(splitInteger(8, 1)).toEqual([8]);
 });
 
 test('should sort parts ascending if they are not equal', () => {
-
+  expect(splitInteger(12, 3)).toEqual([4, 4, 4]);
 });
 
 test('should add zeros if value < numberOfParts', () => {
-
+  expect(splitInteger(6, 6)).toEqual([1, 1, 1, 1, 1, 1]);
```

Comment:
`should add zeros if value < numberOfParts` 

In this test you are passing the value that is equal to numberOfParts.